apps/infrastructure/src/cloudfront.tf

```diff
@@ -132,7 +132,14 @@ module "cloudfront_cms" {
 
   enabled         = true
   is_ipv6_enabled = true
-  comment         = "CloudFront distribution for the medialibrary cms."
+  comment         = "CloudFront distribution for the CMS Media Library"
+
+  viewer_certificate = {
+    cloudfront_default_certificate = false
+    acm_certificate_arn            = module.strapi_media_library_ssl_certificate.acm_certificate_arn
+    ssl_support_method             = "sni-only"
+    minimum_protocol_version       = "TLSv1.2_2021"
```

Comment:
I followed the [AWS Production Security checklist of PagoPA](https://pagopa.atlassian.net/wiki/spaces/DEVOPS/pages/965641850/AWS+Production+Security+checklist).